models/ThoughtModel.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
```

Comment:
Did you know you can put these two imports on the same line? 😍 Since they are both from Mongoose, you can combine the imports, like this for example: 
`import mongoose, { Schema } from "mongoose";`

Just a small tip 😄